herd/machAction.ml

```diff
@@ -487,7 +489,8 @@ end = struct
       List.map
         (fun (tag,p) ->
           let p act = match act with
-          | Access(_,_,_,annot,_,_,_)|Amo (_,_,_,annot,_,_,_) -> p annot
+          | Access(_,_,_,annot,_,_,_)|Amo (_,_,_,annot,_,_,_)
+          | Fault (_,_,_,annot,_)-> p annot
```

Comment:
If I understand correctly, this change will add `Fault` in addition to `Access` and `Amo` to `asets`.

Reply:
Hi @maranget, thanks for reviewing this change!

I added a comment to the line which I think implements the change. But I might be missing something, please let me know if I do.

Reply:
I have read to code too quickly. The code works as advertised, as I checked on some picture with option `-doshow L` for displaying the `L` (Release) set.

Reply:
Thanks, this is the product of many rewritings. If only the whole system was like that :)